mara_db/shell.py

```diff
@@ -351,18 +360,26 @@ def __(db: dbs.RedshiftDB, target_table: str, csv_format: bool = None, skip_head
 @copy_from_stdin_command.register(dbs.BigQueryDB)
 def __(db: dbs.BigQueryDB, target_table: str, csv_format: bool = None, skip_header: bool = None,
        delimiter_char: str = None, quote_char: str = None, null_value_string: str = None, timezone: str = None):
+    assert db.gcloud_gcs_bucket_name, f"Please provide the 'gcloud_gcs_bucket_name' parameter to database '{db}' "
```

Comment:
This one probably also needs a changelog entry, although I don't really understand how it could be used without a bucket?

Reply:
Yes, unfortunately the bq command line tool dropped support for copying from stdin

Reply:
Did you try to pipe with filename `/dev/stdin`? This is probably not officially supported but might work. I used this hack in #39 to make piping work with sqlcmd. Nevertheless, I would recommend everybody to use the GCS bucket if possible.

Reply:
this is just an idea to make it work. But I don't have the need for it in my environment.

Reply:
this is just an idea to make it work. But I don't have the need for it in my environment.